utilities/dict_utils.py

```diff
@@ -0,0 +1,6 @@
+def modified_pop(dictionary, key):
+    return {
+        k: dictionary[k] 
+        for k in dictionary 

```

Comment:
Ah! Sorry, so basically in python if we do dictionary.pop(key), it returns the popped number.

so if this is my dictionary
```python
my_dict = {
    'key1':'value1',
    'key2':'value2'
}
```
my_dict.pop('key1') will return 'value1'. What I wanted, was instead 
```
{
    'key2':'value2'
}
```

So that way, I get everything EXCEPT the id that I just removed :) .